bindings/java/CMakeLists.txt

```diff
@@ -229,6 +229,7 @@ if(NOT OPEN_FOR_IDE)
     COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:fdb_java> ${lib_destination} &&
     ${CMAKE_COMMAND} -E copy $<TARGET_FILE:fdb_java> ${jni_package} &&
     ${CMAKE_COMMAND} -E touch ${CMAKE_CURRENT_BINARY_DIR}/lib_copied
+    DEPENDS fdb_java
```

Comment:
I don't think this is correct -- I though `DEPENDS` is meant to list files. Instead it should be in `add_dependencies` below...